src/main/java/jdbchomework/console/CustomersMenu.java

```diff
@@ -0,0 +1,65 @@
+package jdbchomework.console;
+
+import jdbchomework.controller.CustomerController;
+import org.slf4j.LoggerFactory;
+
+public class CustomersMenu extends AbstractMenu {
+
+    private static final String CUSTOMER_NOT_FOUND = "Customer not found";
+    private static final String PLEASE_ENTER_CUSTOMER_ID = "Please enter customer ID: ";
+    private static final String PLEASE_ENTER_CUSTOMER_NAME = "Please enter customer name: ";
+
+    private CustomerController customerController;
+
+    public CustomersMenu(MenuContents menuContents, CustomerController customerController) {
+        super(LoggerFactory.getLogger(CustomersMenu.class), menuContents);
+        this.menuContents = menuContents;
+        this.customerController = customerController;
+    }
+
+    public void menu() {
+        mark:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Refactor the code to remove this label and the need for it. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ALabelsShouldNotBeUsedCheck)